container/apptabs.go

```diff
@@ -389,8 +393,12 @@ func (r *appTabsRenderer) buildTabButtons(count int) *fyne.Container {
 
 func (r *appTabsRenderer) updateIndicator(animate bool) {
 	if r.appTabs.current < 0 {
+		r.divider.Hide()
```

Comment:
Are you sure the divider should be hidden?
Just because there are no tabs does not mean that this widget becomes invisible. Won't the MinSize still leave room for it?

Reply:
@andydotxyz  I am pretty sure this divider should be hidden, show you the picture, it looks better after hidden the divider.
![divide](https://github.com/fyne-io/fyne/assets/14317611/a0d21f3e-42aa-4e83-9633-1d301bf11d66)


And the tabs minSize calculation does not contains divider, so i think it is ok for doing this.
https://github.com/fyne-io/fyne/blob/79968626fe10c3cd49b978bc2f0d635bc8e18d31/container/tabs.go#L367-L399

Reply:
No other widgets exhibit the behaviour of appearing invisible if they have no content. I wonder what the use-case is for the bar disappearing if items are removed?

Reply:
I agree it looks much better with the bar hidden, but my question is why would you ever have no tabs in this widget? In the case of not having tabs, it should probably be up to the user to hide the apptabs widget I think.

Reply:
Yeah, no tabs in a tab container seems a bit strange. Most applications that I know of close the whole window if you close the last tab. That, or just hiding the tab widget itself if necessary, seems like something the developers can do already

Reply:
@spxvszero Okay yeah, got it. I think in that case you'd either hide or not show the apptabs when there are none.

Reply:
Any chance you were able to finish this up @spxvszero ? Sorry to bug, but would really like to see this get merged in as the current bug on develop looks terrible. 

I guess I could just use your fork instead. lol

Reply:
It could be wise to separate out the bug fix and the behaviour change. I think it should make this land more quickly

Reply:
@andydotxyz @ismyhc @Jacalz Sorry for replying so late. I think we cannot figure out all the use case and stand for their perspectives. But we can make it looks more natural and comfortable, so i did these change. I agree with @Jacalz , i should separate out the bug fix and the behavior change, i discard the behavior change.Thanks for reviewing.